EpikCord/__init__.py

```diff
@@ -529,8 +530,22 @@ def figure_out_interaction_class():
             if bool(command_exists): # bool(Filter) returns True every time.
                 if interaction.options:
                     for option in interaction.options:
-                        option_values.append(option.get("value"))
-                await command_exists[0].callback(interaction, *option_values)
+                       option_values.append(option.get("value"))
+                total_checks = len(command_exists[0].check)
+                checks_completed = 0
+                for check in command_exists[0].check:
+                    if asyncio.iscoroutinefunction(check):
+                        checked = await check(interaction)
```

Comment:
What value will `check` return for it to be truthy, just wondering because if they just put in `return`...

Reply:
boolean

Reply:
But the issue is if someone adds a custom check, and they don't even have a return surely that'd be an issue

Reply:
bruh, that can easily be solved by documentation

Reply:
Help channels will still be flooded.

Reply:
Hmm, will look into it